src/componentLibrary/CTA/CTALink.tsx

```diff
@@ -10,11 +11,11 @@ interface CTALinkProps extends React.ComponentPropsWithRef<'a'> {
 const CTALink = forwardRef<HTMLAnchorElement, CTALinkProps>(
   ({ variant, fullWidth = false, className, href, children, ...rest }, ref): JSX.Element =>
```

Comment:
Should we still pick `href` property and pass it to the CTA component? I think we could remove it now that we use the `Link` from react-router :)

Reply:
No, we should get rid of that :)

Reply:
Done